packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -2912,20 +2918,41 @@ Iterable<DiagnosticsNode> _describeRelevantUserCode(Element element) {
       ErrorSpacer(),
     ];
   }
+
+  bool isOverflowError() {
+    for (final DiagnosticsNode node in properties) {
+      if (node is ErrorSummary) {
+        if (node.value.isNotEmpty
+            && node.value.first is String
+            && (node.value.first as String).contains('A RenderFlex overflowed by')) {
+          return true;
+        }
+      }
+    }
+    return false;
+  }
+
   final List<DiagnosticsNode> nodes = <DiagnosticsNode>[];
   bool processElement(Element target) {
     // TODO(chunhtai): should print out all the widgets that are about to cross
     // package boundaries.
     if (debugIsLocalCreationLocation(target)) {
 
+
       DiagnosticsNode? devToolsDiagnostic;
-      final String? devToolsInspectorUri =
-          WidgetInspectorService.instance._devToolsInspectorUriForElement(target);
-      if (devToolsInspectorUri != null) {
-        devToolsDiagnostic = DevToolsDeepLinkProperty(
-          'To inspect this widget in Flutter DevTools, visit: $devToolsInspectorUri',
-          devToolsInspectorUri,
-        );
+
+      // TODO(kenz): once the inspector is better at dealing with broken trees,
+      // we can enable deep links for more errors than just RenderFlex overflow
+      // errors.
```

Comment:
file a bug for this. Filtering to only show one error is probably ok for now to incrementally roll out this feature but it would be a problem if this code stays in place long term.

Reply:
done